code/modules/clothing/head/_head.dm

```diff
@@ -12,7 +12,9 @@
 	var/light_applied
 	var/brightness_on
 	var/on = 0
-
+	origin_tech = "{'materials':1,'engineering':1}"
+	material = /decl/material/solid/cloth 
```

Comment:
`origin_tech` and `material` are already set to these values on `/obj/item/clothing`, they don't need to be set again.

Reply:
Ill changed this out later